docs/get-started.md

```diff
@@ -24,9 +24,12 @@ If you have not signed up before, you can do so [here](https://improbable.io/get
 
 - Make sure you download the **Installer** version, and select the following components during installation:
     - **Linux Build Support**
-    - **Mac Build Support**
+    - **Mac Build Support** 
 
-> **Note:** You need Linux build support because all server-workers in a cloud deployment run in a Linux environment. 
+> **Note:** 
+You need these build supports even though you are developing on a Windows PC.<br/>
+You need **Linux Build Support** because all server-workers in a cloud deployment run in a Linux environment. <br/>
```

Comment:
Minor point, but the capitalisation and bolding are inconsistent (elsewhere, only the OS is bold, and "Build Support" is lower case).